src/new_hope/week2.clj

```diff
@@ -0,0 +1,185 @@
+(ns new-hope.week2
+  (:require [midje.sweet :refer :all])
+  (:require [clojure.test :refer :all])
+  (:require [clojure.set :refer :all])
+  )
+
+
+(defn fibonacci
+  "calculates fibonacci sequence up to the given argument"
+  [length]
+  (take length (map first (iterate (fn [[x y]] [y (+ x y)]) [1 1]))))
+
+;Day 1
+(deftest test-fibonacci-sequence
+  (is (= (fibonacci 3) '(1 1 2)))
+  (is (= (fibonacci 6) '(1 1 2 3 5 8)))
+  (is (= (fibonacci 8) '(1 1 2 3 5 8 13 21)))
+)
+
+(defn get-the-caps
+  "returns only uppercase characters of the argument sequence"
+  [x]
+  (filter #(Character/isUpperCase %) x)
+)
+
+(deftest test-only-uppercase
+  (is (get-the-caps "HeLlO, WoRlD!") "HLOWRD")
```

Comment:
you forgot (= ...) here, this will always succeed
should be:
`(is (= (get-the-caps "HeLlO, WoRlD!") "HLOWRD"))`

and below as well